modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -364,10 +364,16 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
     return keychains;
   }
 
-  async createRShareFromTxRequest(params: {
+  async createCommitmentShareFromTxRequest(params: {
     txRequest: TxRequest;
     prv: string;
-  }): Promise<{ rShare: SignShare; signingKeyYShare: YShare }> {
+    walletPassphrase: string;
+  }): Promise<{
+    userToBitgoCommitment: CommitmentShareRecord;
+    encryptedSignerShare: EncryptedSignerShareRecord;
+    encryptedUserToBitgoRShare: EncryptedSignerShareRecord;
+  }> {
+    const bitgoIndex = 3;
```

Comment:
im not sure but i think there are some defined constants in the SDK for the indices of bitgo, user, and backup. We could make use of here, instead of directly using 3.